src/components/Tabs/TabNavItems.js

```diff
@@ -0,0 +1,28 @@
+import React, { useState } from "react";
+
+import "./Tabs.css";
+
+const TabNavItems = ({ data, value, handleOnClick, id }) => {
+  const allyears = data.map((item) => new Date(item.date).getFullYear());
+  const years = allyears
+    .filter(function (item, position) {
+      return allyears.indexOf(item) === position;
+    })
+    .sort()
+    .reverse();
+
+  const [activeIndex, setActiveIndex] = useState(0);
```

Comment:
This seems doing the same role with `value` in `Tabs` component and it's even passed to this component. Maybe we can use that here as well instead of creating a new state?